add-to-org.rb

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env ruby
+# add USERNAME as member to ORGANIZATION and optionally to TEAM
+# assumes you created and stored the appropriate OAuth token as an ENV variable called GITHUBUSER_TOKEN
+
+require 'octokit'
+require 'optparse'
+
+# Make sure arguments are specified
+ARGV << '-h' if ARGV.empty?
+
+# Create options hash
+options = {}
+
+# Parse options
+OptionParser.new do |opts|
+  opts.banner = "Usage: add-to-org.rb [options]"
+
+  opts.on("-u", "--username USERNAME", "Username for membership -- ex: githubstudent") do |u|
+    options[:username] = u
+  end
+
+  opts.on("-o", "--org ORGANIZATION", "Organization name -- ex: githubschool") do |o|
+    options[:org] = o
+  end
+
+  opts.on("-t", "--team TEAM_NAME", "[Optional] Team to add member to -- ex: developers") do |t|
+    options[:team] = t
+  end
+
+  opts.on_tail("-h", "--help", "Prints this help message") do |h|
+    puts opts
+    exit
+  end
+
+end.parse!
+
+
+TOKEN = ENV['GITHUBTEACHER_TOKEN']
+abort("Missing GITHUBUSER_TOKEN. Please set up an OAUTH token and set it in the environment by typing\n
+export GITHUBTEACHER_TOKEN=XXXXXXXXXXXXXXXXXXXXXXX\n
+and replace the Xs with your actual token. Reminder: This token needs admin privilages onto your organization in order to be inviting people.") unless TOKEN
+
+# Assign variables
+#  Team is optional, so you don't need to assign it if it's nil.
+username = options[:username]
+org = options[:org]
+team_name = options[:team] if !options[:team].nil?
+
+
+# Create a new Octokit Client
+Octokit.auto_paginate = true
+@client = Octokit::Client.new :access_token => TOKEN
+
+# If the team doesn't exist yet, create it
+#  otherwise, get the team_id from the list
+# Once we have the team_id we can add people to it
+#  regardless of them being a member of the organization yet
+def add_to_team_and_org(team_name, username, org)
+
+  team_id = get_team_id(team_name, org)
+  create_team(org, team_name, org) unless !team_id.nil?
+
+  # Add username to team_id
+  # Slightly different than adding them as a member.
+  # If the username includes ","s, split it because it's a full list!
+  # TODO: break this out into it's own method detected early on?
+  begin
+    if username.include?(",")
```

Comment:
You can `username.split(",")` and it will always return an array, even if `username` has no delimiters, so the conditional is superfluous. You can get rid of it and the `else` case.

**e.g.**
```
irb(main):001:0> "abc,def".split(',')
=> ["abc", "def"]
irb(main):002:0> "abc".split(',')
=> ["abc"]
irb(main):003:0> "".split(",")
=> []
```